.github/workflows/examples.yml

```diff
@@ -2,7 +2,6 @@ name: Examples
 
 on:
   pull_request:
-    branches: [ master ]
```

Comment:
For example, this action is not executed on a PR targeted to `branch-2.0`.

Reply:
Shall we limit CI run on active branch ? (e.g master and branch-2.0), other branch there's no commits.

Reply:
We already do for `push`.